lang/compiler.go

```diff
@@ -84,11 +84,14 @@ func Verbose(v bool) CompileOption {
 
 // Compile evaluates a Flux script producing a flux.Program.
 // now parameter must be non-zero, that is the default now time should be set before compiling.
-func Compile(q string, now time.Time, opts ...CompileOption) (*AstProgram, error) {
+func Compile(extern *ast.File, q string, now time.Time, opts ...CompileOption) (*AstProgram, error) {
```

Comment:
Can the extern values get added as a compile option? It seems weird to make it the first parameter and required when it is optional.

Reply:
Yes, good idea.  I repushed.